lemarche/utils/apis/api_brevo.py

```diff
@@ -28,26 +29,43 @@ def get_api_client():
     return sib_api_v3_sdk.ApiClient(config)
 
 
-def create_contact(user, list_id: int):
+def create_contact(user, list_id: int, tender=None):
     """
     Brevo docs
     - Python library: https://github.com/sendinblue/APIv3-python-library/blob/master/docs/CreateContact.md
     - API: https://developers.brevo.com/reference/createcontact
     """
     api_client = get_api_client()
     api_instance = sib_api_v3_sdk.ContactsApi(api_client)
+
+    attributes = {
+        "NOM": sanitize_to_send_by_email(user.last_name.capitalize()),
+        "PRENOM": sanitize_to_send_by_email(user.first_name.capitalize()),
+        "DATE_INSCRIPTION": user.created_at,
+        "TYPE_ORGANISATION": user.buyer_kind_detail,
+        "NOM_ENTREPRISE": sanitize_to_send_by_email(user.company_name.capitalize()),
+        "SMS": sanitize_to_send_by_email(user.phone_display),
+        "MONTANT_BESOIN_ACHETEUR": None,
+        "TYPE_BESOIN_ACHETEUR": None,
+        "TYPE_VERTICALE_ACHETEUR": None,
+        # WHATSAPP, TYPE_ORGANISATION, LIEN_FICHE_COMMERCIALE, TAUX_DE_COMPLETION
+    }
+
+    if tender:
+        sectors = tender.sectors.all()
```

Comment:
```suggestion
```
Pas besoin de cette requête, à la ligne 60, tu peux au lieux du exists, tu check qu'il est pas nul :) 
```suggestion
        first_sector = tender.sectors.first()
```